src/style.css

```diff
@@ -1 +1,97 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 700;
+  font-style: normal;
+}
+
+.paige {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.paige__body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  width: 100%;
+}
+
+.search-bar {
+  width: 100%;
+}
+
+.search-bar__input {
+  width: 100%;
+  outline: none;
+  text-align: left;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-size: 19px 19px;
+}
+
+.search-bar__input.big {
+  min-width: 900px;
+  margin-top: 20px;
+  height: 70px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  background-position: left 26px center;
+}
+
+.search-bar__input.small {
```

Comment:
Similar to the previous comment, setting a fixed 'min-width' here can cause layout issues on smaller screens. It's better to use responsive design techniques to ensure the input field scales appropriately.